packages/eslint-plugin/src/util/getESLintCoreRule.ts

```diff
@@ -6,6 +6,7 @@ const isESLintV8 = semver.major(version) >= 8;
 
 interface RuleMap {
   /* eslint-disable @typescript-eslint/consistent-type-imports -- more concise to use inline imports */
+  'consistent-return': typeof import('eslint/lib/rules/consistent-return');
```

Comment:
[Style] Nit: these are pretty much all in alphabetical order at the moment. Up for humoring me and inserting this in sorted order? 😁